scripts/perftune.py

```diff
@@ -1351,7 +1348,7 @@ def dump_config(prog_args):
     if prog_args.mode:
         prog_options['mode'] = prog_args.mode
 
-    if prog_args.nic:
+    if prog_args.nics:
         prog_options['nic'] = prog_args.nics
```

Comment:
What is this `prog_options` used for? Why are we now setting `prog_args.nics` to `nic` key`? Shouldn't that be `nics`?

Reply:
this would change the perftune.yaml file and we don't want it
the logic behind this is the same as with --dir option
so basically a safety check here is to compare this parsing and dumping of "nic"
is the same as with "dir"